packages/directory/addon/services/directories.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright 2019, Yahoo Holdings Inc.
+ * Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms.
+ */
+import Service from '@ember/service';
```

Comment:
Is this more of an util rather than a service?

Reply:
It is mostly a util, but the static file of directories that it depends on is harder to override in an app without overriding the entire util. As a service, the app just needs to extend this service with their own directories.

Reply:
Should we just move the enums under here then as a default for directories? Keep it closer to this service?

Reply:
Okay I've moved the default directories into the service so now the utils file isn't needed